build-system/tasks/check-links.js

```diff
@@ -143,6 +143,10 @@ function filterWhitelistedLinks(markdown) {
   // Links inside a <code> block (illustrative, and not always valid)
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
 
+  // The heroku nightly build page is not always acccessible by the checker.
+  filteredMarkdown = filteredMarkdown.replace(
+      /\(http:\/\/amphtml-nightly.herokuapp.com\/\)/g, '');
```

Comment:
Replace `http` with `https?`

Reply:
The original URL in `DEVELOPING.md` is `http`, so we must whitelist it that way.

Reply:
`https?` (question mark included) means it'll catch both `http` and `https`, in case someone changes the documentation later :)

Reply:
Aha! Right now, we only have the one place where we document the URL, so I'll keep it this way for simplicity.